storagemodule.py

```diff
@@ -0,0 +1,57 @@
+#Question: Do you want me to make this whole thing a module or lots of submodules ? 
+
+#Import libaries
```

Comment:
You can delete this comment, as it's pretty obvious what this is. (I really dislike that they say you should comment code. It's a lie. Code should be self explanatory with proper variable and function naming. A comment should only ever be used if you needed to do a hack of some kind because of some technical limitation - which is basically never.